ovs/extensions/migration/migration/ovsmigrator.py

```diff
@@ -66,6 +66,8 @@ def migrate(previous_version, master_ips=None, extra_ips=None):
             mds_safety_key = '/ovs/framework/storagedriver'
             if Configuration.exists(key=mds_safety_key):
                 current_mds_settings = Configuration.get(key=mds_safety_key)
+                if 'mds_tlogs' in current_mds_settings:
```

Comment:
Reason to popping here?

Reply:
mds_tlogs is configurable via the GUI on the StorageDriver, so no need for this value in our config mgmt, since its available on SD level